docs/android/v2/advanced-features/hls-timed-metadata.mdx

```diff
@@ -0,0 +1,48 @@
+---
+ title: HLS Timed Metadata
+ nav: 8.05
+---
+
+HLS Timed Metadata feature helps you synchronise certain events with the HLS stream. This can be useful for showing interactive quizzes / product overlays etc.
+
+## Requirements
+
+- 100ms Android SDK version 2.5.6 or higher
+- Active HLS stream
+
+## Sending HLS Timed Metadata
+
+To add HLS timed metadata cue to the currently running HLS stream use `setHlsSessionMetadata` API like this:
+
+```kotlin
+val metadata = HMSHLSTimedMetadata(
+                        payload  = payloadText,
+                        duration = durationText
+                    )
+
+hmsSDK.setHlsSessionMetadata(arrayListOf(metaDataModel), object : HMSActionResultListener {
```

Comment:
is this correct? seeing `SessionMetadata`

Reply:
`SessionMetadata` where did you see this ?